.prettierignore

```diff
@@ -1,2 +1,13 @@
 node_modules
 public
+src/types/*
+src/serviceWorker.ts
```

Comment:
Why this file is not included?

Reply:
I was under the impression serviceWorker.ts was a generated file by another package so didnt want to touch it. Happy to remove it thought if you think it should be checked.

Reply:
Ohh, ok.In that case I think we can leave it.